demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -584,6 +584,32 @@ def test_generate_integration_doc(self):
         assert "Number of users to return. Max 300. Default is 30." in open(fake_readme).read()
 
 
+def test_get_command_examples(tmp_path):
```

Comment:
add a unit test with commands not starting with `!`

Reply:
done.